src/pages/Browse/index.spec.tsx

```diff
@@ -84,6 +85,57 @@ describe(__filename, () => {
     );
   };
 
+  const setUpVersionFileUpdate = ({
+    extraFileEntries = {},
+    initialPath = 'manifest.json',
+    loadVersionAndFile = true,
+  } = {}) => {
+    const addonId = 9876;
+    const version = {
+      ...fakeVersion,
+      id: 1234,
+      file: {
+        ...fakeVersion.file,
+        entries: {
+          ...fakeVersion.file.entries,
+          ...extraFileEntries,
+        },
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        selected_file: initialPath,
+      },
+    };
+    const store = configureStore();
+
+    if (loadVersionAndFile) {
+      _loadVersionAndFile({ store, version });
+    }
+
+    const fakeThunk = createFakeThunk();
+    const _fetchVersionFile = fakeThunk.createThunk;
+
+    return {
+      _fetchVersionFile,
+      addonId,
+      fakeThunk,
+      store,
+      updateProps: (props = {}) => {
+        const dispatchSpy = spyOn(store, 'dispatch');
```

Comment:
We tend to use `dispatch` without the `Spy` suffix.

Reply:
In this case, I think it would be confusing to call this variable `dispatch` because one might think you could dispatch actions with it. In other cases it's probably OK but when using this helper, it's not obvious that this is a spy object (the `spyOn()` call is hidden).